pkg/app/piped/cmd/piped/piped.go

```diff
@@ -327,6 +328,18 @@ func (p *piped) run(ctx context.Context, t cli.Telemetry) (runErr error) {
 		})
 	}
 
+	// Start running image watcher.
+	{
+		t := imagewatcher.NewWatcher(
```

Comment:
This should be initialized and started only if there is at least one image provider specified.

Reply:
Right.

Reply:
then back to this comment https://github.com/pipe-cd/pipe/pull/1282#discussion_r545541692, once it been initialized, not found configuration file should be an error 🤔

Reply:
If so I feel like it should be the INFO severity. It's tough to go into it, but the significant thing is that this `ImageWatcher` config file is maintained by a user (not a piped operator).

https://github.com/pipe-cd/pipe/blob/219102c0bdf983c30a3a82b81f885e41f8785d04/pkg/config/testdata/.pipe/image-watcher.yaml#L1-L6

And the `imageProvider` mentioned by @nghialv is maintained by a piped operator.

https://github.com/pipe-cd/pipe/blob/7e2e9aa05cbce41f26c33cdacdd25c514670ce67/pkg/config/testdata/piped/piped-config.yaml#L80-L85

Hence, it could be a general case that `imageProvider` is configured but `ImageWatcher` is not configured, that's why I made it INFO.

Reply:
No need to restart piped to make this log be gone, just add an image-watcher file to the config-repo. (Sorry, I know this is not enough description 😓

Reply:
got it 🙆‍♀️ thank you 👍